src/config_logs/google-data-transfer/googlge-data-transfer.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Why you didn't add a storage_transfer_job entity log linking despite avilable logs in GPC?
![image](https://github.com/dynatrace-oss/dynatrace-gcp-monitor/assets/138470854/f5992639-dc3f-425a-a106-a1d2ef975472)